src/components/Article/ArticleController.tsx

```diff
@@ -52,8 +41,10 @@ const ArticleController = (props: Props) => {
     question: '',
     answer: '',
     newCategory: '',
+    searchCriteria: '',
   });
-
+  const [items, setItems] = useState<undefined | any[]>([{}]);
+  const [list, setList] = useState<undefined | any[]>([{}]);
```

Comment:
remove list. use local items to store filtered data from props.article.items

- props.article.items = source data with no filters
- items = filtered based on search or same as source when no search parameter is provided

Reply:
Modified the code as per the comments